crates/autopilot/src/run_loop.rs

```diff
@@ -647,7 +647,10 @@ struct Metrics {
     auction: prometheus::IntGauge,
 
     /// Tracks the duration of successful driver `/solve` requests.
-    #[metric(labels("driver", "result"))]
+    #[metric(
+        labels("driver", "result"),
+        buckets(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15)
```

Comment:
I think the current time limit is 17, so maybe make the buckets go to 20 just to be safe?